source/facter/3.0/core_facts.md

```diff
@@ -4,20 +4,19 @@ title: "Facter 3.0: Core Facts"
 toc: columns
 ---
 
-
 This is a list of all of the built-in facts that ship with Facter 3.0.
```

Comment:
Since this is the introductory line of text, maybe change this line to call out the distinction from the get-go. E.g. something like:

"Above is a list of all the built-in facts that ship with Facter 3.0. Please note that built-in facts include both legacy facts (which are deprecated) and newer structured facts."

Note also that I used the term "legacy" above rather than "unstructured." I'm not totally wedded to that, but here's why I phrased it that way (and btw: pedant alert): there are one or two built-in facts which are both legacy and structured, i.e. because they were introduced in facter 2 (when we first added structured facts) but then restructured in facter 3 (when we structured *everything* and thought the structure through systematically).

In short, "legacy" is a bit more accurate than "unstructured" and may also connote "stop using this" more. It also matches the `--show-legacy` command line option. But I'm terrible at naming things :)

If we do opt for "legacy" here, let's make that change in the per-fact lines modified below as well.